dvc/repo/experiments/ls.py

```diff
@@ -39,6 +43,11 @@ def ls(
         name = baseline[:7]
         if tags[baseline] or ref_heads[baseline]:
             name = tags[baseline] or ref_heads[baseline][len(base) + 1 :]
-        results[name] = [info.name for info in ref_info_dict[baseline]]
+        for info in ref_info_dict[baseline]:
+            if git_remote:
+                exp_rev = None
```

Comment:
Not sure if there's an easy way to get the rev of a remote ref. Skipped it here.

Reply:
This is possible to do but it's not implemented right now in scmrepo, we can add it in the future if it's actually needed